CheckerServer/CheckerServer/Models/OrderItem.cs

```diff
@@ -28,7 +28,7 @@ public enum ItemStatus      // the status of the order item overall
         Returned                // customer returned item for some reason
     }
 
-    public enum LineItemStatus  // status of the order item in the line of preperation 
+    public enum eLineItemStatus  // status of the order item in the line of preperation 
     {
         Locked,
```

Comment:
Available instead of To-Do

Reply:
As this is meant to reflect the list it belongs to in the Line, ( 1 of 3 lists), Keeping the name the same as the list name is meaningful.